docs/contributing/how-to-run-website-locally.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "How to run the website locally"
+linkTitle: "How to run the website locally"
+weight: 2
+slug: how-to-run-the-website-locally
+---
+
+The Cortex documentation is compiled into a website published at [cortexmetrics.io](https://cortexmetrics.io/). This document explains how to run the website locally, in order to have a quick feedback loop while contributing to the documentation or website styling.
+
+
+## Initial setup
+
+The following initial setup is required only once:
+
+1. Install [Hugo](https://gohugo.io/) `v0.59.1`
+2. Install [Node.js](https://nodejs.org/en/) (or via [`nvm`](https://github.com/nvm-sh/nvm))
+3. Install required Node modules
+   ```
+   cd website && npm install -g postcss-cli autoprefixer
```

Comment:
Do these need to be global? I don't understand the full process but I think these should be locally installed to be picked up by the build process (as you might not have access via the global install path). Since you already have a `package.json` on the root of `website` I would suggest using `npm install` as the comand to run. 


The error I received was:

```
Error: Loading PostCSS Plugin failed: Cannot find module 'autoprefixer'
```

Which has you can see means it fails to find the dependency.